cmd/antrea-agent-simulator/main.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main under directory cmd parses and validates user input,
+// instantiates and initializes objects imported from pkg, and runs
+// the process.
```

Comment:
I am not sure this is useful. If you want to add a package-level comment, maybe take a look at the one for kubelet: https://github.com/kubernetes/kubernetes/blob/564ffbd400e50f37b892747c434ec55bde865134/cmd/kubelet/kubelet.go#L17-L20. You want to be describing what the binary is for.

Reply:
done